### DISCUSSION TITLE
How to get the #select values in csv file

### DISCUSSION THREAD

**Discussion by kal-purush** [github/securitylab#704] [Q&A]
How is to possible to get the query result shown in vs code query result in CLI? If I run the same query in CLI and save the result in csv file, I got different result. 

<img width="362" alt="Screenshot 2022-09-13 at 14 44 25" src="https://user-images.githubusercontent.com/88188566/189904310-1b25eabf-b895-49be-aa5e-6cbf2408bb07.png">

this is the result from vs code, but the same query produces this result in csv file

<img width="724" alt="Screenshot 2022-09-13 at 14 45 57" src="https://user-images.githubusercontent.com/88188566/189904541-cb5f3c2f-b48b-4894-89f6-c8a015b9ea16.png">

how can I get the vs code result in CSV file?

**Comment by MathiasVP** [comment]
Hi @kal-purush,

Can you show me:
- the query you're trying to get the output from.
- The CLI command you're running to get the csv output.

The reason I'm asking for these things is that CodeQL queries have some meta-data at the top of the file that tells the CLI how to interpret the results. The VSCode extension does this automagically for you when this meta-data is present, but if you're calling the CLI directly you might need to do that manually (depending on how you invoke the CLI).
